streams/readable-byte-streams/detached-buffers.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
+
+if (self.importScripts) {
+  self.importScripts('/resources/testharness.js');
+}
+
+promise_test(() => {
+  const stream = new ReadableStream({
+    start(c) {
+      c.close();
+    },
+    type: 'bytes'
+  });
+
+  const reader = stream.getReader({ mode: 'byob' });
+  const view = new Uint8Array([1, 2, 3]);
+  return reader.read(view).then(({ value, done }) => {
+    // Sanity checks
+    assert_true(value instanceof Uint8Array);
```

Comment:
I realise that these are not expected to fail, but if they do it will be very annoying that they don't have messages associated. If this behaviour is adequately tested elsewhere, maybe just take the asserts out?